src/ray/core_worker/core_worker.h

```diff
@@ -0,0 +1,62 @@
+#ifndef RAY_CORE_WORKER_CORE_WORKER_H
+#define RAY_CORE_WORKER_CORE_WORKER_H
+
+#include "common.h"
+#include "object_interface.h"
+#include "ray/common/buffer.h"
+#include "task_execution.h"
+#include "task_interface.h"
+
+namespace ray {
+
+/// The root class that contains all the core and language-independent functionalities
+/// of the worker. This class is supposed to be used to implement app-language (Java,
+/// Python, etc) workers.
+class CoreWorker {
+ public:
+  /// Construct a CoreWorker instance.
+  ///
+  /// \param[in] worker_type Type of this worker.
+  /// \param[in] langauge Language of this worker.
+  CoreWorker(const WorkerType worker_type, const Language language)
```

Comment:
Since `worker_type` and `language` are passed by value, it's unnecessary to declare them as `const`.

Reply:
there's a minor difference. using `const` can prevent the argument from being modified in the function.